backend/src/kitconcept/intranet/profiles/default/types/Person.xml

```diff
@@ -5,8 +5,22 @@
         i18n:domain="kitconcept.intranet"
 >
   <property name="behaviors"
-            purge="false"
+            purge="true"
   >
+    <element value="kitconcept.core.person_image" />
+    <element value="volto.kicker" />
+    <element value="collective.person.person" />
+    <element value="kitconcept.intranet.person" />
+    <element value="collective.contact_behaviors.contact_info" />
+    <element value="kitconcept.core.additional_contact_info" />
+    <element value="plone.namefromtitle" />
+    <element value="plone.shortname" />
+    <element value="volto.navtitle" />
+    <element value="plone.excludefromnavigation" />
+    <element value="plone.relateditems" />
+    <element value="plone.versioning" />
+    <element value="plone.locking" />
+    <element value="plone.translatable" />
```

Comment:
Have we considered which fields should be language-independent?

Reply:
Also, coming from `kitconcept.core`, we should move this discussion there